app/browser/windows.js

```diff
@@ -72,19 +75,26 @@ const siteMatchesTab = (site, tab) => {
   return matchesLocation && matchesPartition
 }
 
-const updatePinnedTabs = (win) => {
+const updatePinnedTabs = (win, appState) => {
```

Comment:
not sure why we are passing state in, which we get in the same way as we did before?

Reply:
Seems like a better practice and more consistent with the way other functions are structured. Plus it avoids calling AppStore.getstate multiple times in a row for multiple windows (though yes I know it is a very cheap function) and it feels cleaner and easier to test. Plus I’m not sure at all why the function had the AppStore require inside the function before...?? Was it to stop the module from being evaluated in case it wasn’t ever required in the process (I can’t imagine that happening)